app/utils/naming.py

```
@@ -140,10 +146,39 @@ def build_release_name(
+        f"Building release name for series_title={series_title}, season={season}, episode={episode}, height={height}, vcodec={vcodec}, language={language}, site={site}"
     )
+
+    # Use site-specific release group if available
+    try:
+        from app.config import CATALOG_SITE_CONFIGS  # local import to avoid cycles
+
+        release_group = CATALOG_SITE_CONFIGS.get(site, {}).get(
+            "release_group", release_group
+        )
+    except Exception:
+        pass
```

Ruff (0.14.1)

`try`-`except`-`pass` detected, consider logging the exception

(S110)